File: src/main/java/arc/arc/sync/base/Sync.java
package arc.arc.sync.base;

import org.bukkit.entity.Player;
import org.bukkit.event.Event;

import java.util.UUID;

public interface Sync {

    default void forceSave(UUID uuid){}
    default void playerJoin(UUID uuid) {}
    default void playerQuit(UUID uuid) {}
    default void processEvent(Event event) {}

}

File: src/main/kotlin/ru/arc/sync/base/Sync.kt
package ru.arc.sync.base

import org.bukkit.event.Event
import java.util.UUID

interface Sync {
    fun forceSave(uuid: UUID) {}
    fun playerJoin(uuid: UUID) {}
    fun playerQuit(uuid: UUID) {}
    fun processEvent(event: Event) {}
}
